src/components/FileImporter.css

```diff
@@ -0,0 +1,74 @@
+.file-import {
+    width: 22.5rem;
+    flex-grow: 1;
+    border: dashed 0.25rem var(--nimiq-light-blue);
+    border-radius: 1rem;
+    cursor: pointer;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+    text-align: center;
+    padding-bottom: 5rem;
+    padding-top: 7.5rem;
+    position: relative;
+    background-color: white;
+    transition: background-color 300ms;
+}
+
+.file-import:hover {
+    background-color: var(--nimiq-gray);
+}
+
+.file-import.drag-over {
+    background-color: var(--nimiq-light-blue);
+}
+
+.file-import h3 {
+    line-height: 1.375;
+    margin: 0 auto;
+    max-width: 15rem;
+    font-weight: 600;
+    transition: color 300ms;
+    pointer-events: none;
+}
+
+.file-import.drag-over h3 {
+    color: white;
+}
+
+.file-import .error-message {
+    font-size: 2rem;
+    color: var(--nimiq-red);
+    pointer-events: none;
+}
+
+.file-import input {
+    position: absolute;
+    top: -1000rem; /* out of the viewport */
```

Comment:
From https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file: 
"Note: opacity is used to hide the file input instead of visibility: hidden or display: none, because assistive technology interprets the latter two styles to mean the file input isn't interactive."
Not sure though whether that's also an issue when using the `label` technique.

And I would use `vh` instead of `rem` as `vh` would also work for insanely huge screens :D

Reply:
How usable our interface is without touch, but only assistive technologies, is questionable anyway. I made it display:none now.